pkg/client/didexchange/client.go

```diff
@@ -134,10 +144,16 @@ func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*QueryConn
 }
 
 // QueryConnectionByID fetches single connection record for given id
-func (c *Client) QueryConnectionByID(id string) (*QueryConnectionResult, error) {
-	// TODO sample response, to be implemented as part of #226
+func (c *Client) QueryConnectionByID(connectionID string) (*QueryConnectionResult, error) {
```

Comment:
Out of curiosity, why is the Client method "QueryConnectionByID" and the underlying service method the more succinct "Connection"? Will the client have more methods for executing connection lookups?

Reply:
Yes I think the client will more methods for connection lookups

Reply:
(what about the underlying service?)

Reply:
Strange that a method called `QueryConnectionByID` returns `QueryConnectionResult` objects instead of `Connection` objects.

Also, has `QueryConnectionResult` been formally adopted into the `client` package?

Reply:
done